client/src/pages/DashBoardPage.js

```diff
@@ -34,19 +36,23 @@ export default function DashBoardPage() {
         pb: 2,
       }}
     >
-      <Grid container spacing={2}>
-        <DashboardStackedBarChart
-          title='Daily Transactions'
-          data={dailyTransactions}
-        />
-        <DashboardPieChart title='Spending Categories' data={categoriesData} />
-        <Summary />
-        <DashboardEstimatedSpending></DashboardEstimatedSpending>
-      </Grid>
-
-      <AddGoalItem />
-      <GoalsList />
-      <TopTransactionCategories />
+      <RequireAuthentication>
+        <Grid container spacing={2}>
+          <DashboardStackedBarChart
+            title='Daily Transactions'
+            data={dailyTransactions}
+          />
+          <DashboardPieChart
+            title='Spending Categories'
+            data={categoriesData}
+          />
+          <Summary />
+          <DashboardEstimatedSpending></DashboardEstimatedSpending>
```

Comment:
Should we make this <DashboardEstimatedSpending /> instead?

Reply:
Good catch! I think a better way is to change it to `<DashboardEstimatedSpending/>`